src/css/base.scss

```diff
@@ -417,6 +417,10 @@ a:focus {
     direction: rtl;
 }
 
+.dc__ellipsis-left.direction-left {
+    text-align: left;
```

Comment:
can we add this inside dc__ellipsis-left {} in scss syntax?

Reply:
done